### PR TITLE
Build wheels on Windows

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -43,9 +44,12 @@ def build_extensions(self):
                        "-DMS_WIN32",
                        "-mstackrealign"
                        ] if is32bit else ["-DMS_WIN64"]
+        elif self.compiler.compiler_type == "msvc":
+            xtra_link_args = ["/LTCG:OFF"]
```
Comment (nehaljwani) [inline setup.py]: Reason behind turning off LTCG:
```
  fatal error C1047: The object or library file 'C:\Users\nwani\scratch\backports.lzma\bl-dev\Library\lib\liblzma.lib' was created with an older compiler than other objects; rebuild old objects and libraries
  LINK : fatal error LNK1257: code generation failed
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Professional\\VC\\Tools\\MSVC\\14.15.26726\\bin\\HostX86\\x64\\link.exe' failed with exit status 1257
```

**Comment by nehaljwani** [comment]
This code climate thing is really annoying. I'll need assistance in fixing that.

**Comment by peterjc** [comment]
Don't worry about codeclimate - I seem to have accidentally turned that on, and don't care about this specific warning anyway.

**Comment by peterjc** [comment]
I don't know enough about the Windows linking steps to judge - but you clearly do, and it passed the tests 👍

**Pull request merged**